src/Tasks/Microsoft.NET.Build.Tasks/JoinItems.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Linq;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    public class JoinItems : TaskBase
```

Comment:
I was thinking of an item function or existing task that did something similar, but couldn't see one.